extra/completions/alacritty.fish

```diff
@@ -40,6 +31,10 @@ complete -c alacritty \
 complete -c alacritty \
   -l "hold" \
   -d "Remain open after child process exits"
+complete -c alacritty \
+  -s "o" \
+  -l "option" \
+  -d "Override config file options [example: cursor.style=Beam]"
```

Comment:
Oh I missed this last time, but the same thing as the zsh suggestions applies here too of course.

```suggestion
  -d "Override config file options"
```